test/src/main/java/io/strimzi/test/BaseITST.java

```diff
@@ -34,13 +33,36 @@
     private static final Logger LOGGER = LogManager.getLogger(BaseITST.class);
     protected static final String CLUSTER_NAME = "my-cluster";
 
-    public static final KubeClusterResource CLUSTER = new KubeClusterResource();
+    public static final KubeClusterResource CLUSTER = KubeClusterResource.getKubeClusterResource();
+    private static String namespace = CLUSTER.defaultNamespace();
 
-    public static final Config CONFIG = Config.autoConfigure(System.getenv().getOrDefault("TEST_CLUSTER_CONTEXT", null));
-    public static final DefaultKubernetesClient CLIENT = new DefaultKubernetesClient(CONFIG);
+    public static String setNamespace(String futureNamespace) {
```

Comment:
Can we javadoc these methods (`setNamespace`, `cmdKubeClient` and `kubeClient`) to explain how they're related?